Source/WPEFramework/Controller.cpp

```diff
@@ -837,6 +842,8 @@ namespace Plugin {
 
     Core::hresult Controller::Unregister(Exchange::Controller::ILifeTime::INotification* notification)
     {
+        ASSERT(notification != nullptr);
+
```

Comment:
Also here lets report that the Sink was not unregistered as it did not exists.

Reply:
done